storybook/src/video/videoProperties.js

```diff
@@ -0,0 +1,57 @@
+export default {
```

Comment:
In other cases, whenever we feel the need to share sample data across stories, we've been putting it under the `data` folder. (see as examples `comments.data.js` and `lists.data.js`).